.github/workflows/infrastructure.yml

```diff
@@ -26,8 +26,8 @@ jobs:
         with:
             toolchain: nightly-2024-02-06
 
-      - name: Build
-        run: cargo build --release
+      - name: Build forge binary
+        run: cargo build --release --bin forge
 
       - name: Clone Aave Delivery Infrastructure
         run: git clone https://github.com/Moonsong-Labs/aave-delivery-infrastructure.git
```

Comment:
Could you add `--depth=1` and `-b feat/zksync` to the git clone so we avoid cloning the entire repo in CI (saving time)? And also skip `git checkout feat/zksync` later since it's done by git already. Thanks!

Reply:
Asking simply because you are already changing this file with the pr anyways

Reply:
Done